source/extensions/filters/network/mysql_proxy/mysql_codec_clogin_resp.h

```diff
@@ -15,32 +15,25 @@ namespace MySQLProxy {
 enum ClientLoginResponseType { Null, Ok, Err, AuthSwitch, AuthMoreData };
 
 // ClientLoginResponse could be
-// Protocol::OldAuthSwitchRequest, Protocol::AuthSwitchRequest when server want switch auth method
+// Protocol::OldAuthSwitchRequest, Protocol::AuthSwitchRequest when server wants switch auth method
 // or OK_Packet, ERR_Packet when server auth ok or error
 class ClientLoginResponse : public MySQLCodec {
 public:
   ClientLoginResponse() : type_(Null) {}
-  ~ClientLoginResponse() override { cleanup(); }
-
-  ClientLoginResponse(const ClientLoginResponse& other);                // copy constructor
-  ClientLoginResponse(ClientLoginResponse&& other) noexcept;            // move constructor
-  ClientLoginResponse& operator=(const ClientLoginResponse& other);     // copy assignment
-  ClientLoginResponse& operator=(ClientLoginResponse&& other) noexcept; // move assignment
+  ClientLoginResponse(const ClientLoginResponse&);
+  ClientLoginResponse(ClientLoginResponse&&) noexcept;
```

Comment:
Where is move constructor used?

Reply:
Ok, I will remove these codes which are not used.